assets/app/scripts/directives/alerts.js

```diff
@@ -5,7 +5,9 @@ angular.module('openshiftConsole')
     return {
       restrict: 'E',
       scope: {
-        alerts: '='
+        alerts: '=',
+        // If truthy, hide the close button.
+        persistent: '=?'
```

Comment:
dont want to confuse terminology around alerts and toast, we've been saying alerts are "persistent" even if they have a close icon.  maybe just call this "canDismiss" or something